code/modules/mob/living/basic/pets/sloth.dm

```diff
@@ -0,0 +1,98 @@
+GLOBAL_DATUM(cargo_sloth, /mob/living/basic/sloth)
+
+/mob/living/basic/sloth
+	name = "sloth"
+	desc = "An adorable, sleepy creature."
+	icon = 'icons/mob/simple/pets.dmi'
+	icon_state = "sloth"
+	icon_living = "sloth"
+	icon_dead = "sloth_dead"
+
+	speak_emote = list("yawns")
+
+	can_be_held = TRUE
+	held_state = "sloth"
+
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "kicks"
+	response_harm_simple = "kick"
+
+	attack_verb_continuous = "bites"
+	attack_verb_simple = "bite"
+	attack_sound = 'sound/weapons/bite.ogg'
+	attack_vis_effect = ATTACK_EFFECT_BITE
+
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	gold_core_spawnable = FRIENDLY_SPAWN
+
+	melee_damage_lower = 18
+	melee_damage_upper = 18
+	health = 50
+	maxHealth = 50
+	speed = 10 // speed is fucking weird man. they aren't fast though don't worry
+	butcher_results = list(/obj/item/food/meat/slab = 3)
+
+	ai_controller = /datum/ai_controller/basic_controller/sloth
+
+/mob/living/basic/sloth/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/pet_bonus, "slowly smiles!")
+	AddElement(/datum/element/footstep, footstep_type = FOOTSTEP_MOB_CLAW)
+	AddElement(/datum/element/ai_retaliate)
+	AddComponent(/datum/component/tree_climber)
+
+	if(!mapload || isnull(GLOB.cargo_sloth) || !is_station_level(z))
+		return
+
+	// If someone adds non-cargo sloths to maps we'll have a problem but we're fine for now
+	GLOB.cargo_sloth = src
+
+/mob/living/basic/sloth/Destroy()
+	if(GLOB.cargo_sloth == src)
+		GLOB.cargo_sloth = null
+
+	return ..()
+
+/mob/living/basic/sloth/paperwork
+	name = "Paperwork"
+	desc = "Cargo's pet sloth. About as useful as the rest of the techs."
+	gender = MALE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/basic/sloth/citrus
+	name = "Citrus"
+	desc = "Cargo's pet sloth. She's dressed in a horrible sweater."
+	icon_state = "cool_sloth"
+	icon_living = "cool_sloth"
+	icon_dead = "cool_sloth_dead"
+	gender = FEMALE
+	butcher_results = list(/obj/item/toy/spinningtoy = 1)
+	gold_core_spawnable = NO_SPAWN
+
+/// They're really passive in game, so they just wanna get away if you start smacking them. No trees in space from them to use for clawing your eyes out, but they will try if desperate.
+/datum/ai_controller/basic_controller/sloth
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic/ignore_faction,
+		BB_BASIC_MOB_FLEEING = TRUE,
+		BB_FLEE_TARGETTING_DATUM = new /datum/targetting_datum/basic/ignore_faction,
+	)
+
+	ai_traits = STOP_MOVING_WHEN_PULLED
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/target_retaliate/to_flee,
+		/datum/ai_planning_subtree/flee_target/from_flee_key,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree,
```

Comment:
Does this do anything? It seems like sloths run away from people who attack them, and don't otherwise set a target for non-fleeing behaviour.

Reply:
You're still on the shitlist via the retaliate element so I presume something would happen... Might remove it

Reply:
Well what I mean is that if you will always run away from someone on that list it will stop planning there, so it will never reach this beheaviour 😅

Reply:
well doesn't it cancel the behavior if it's impossible to execute (like there's no where to escape to) or am i stupid

Reply:
yeah it doesnt cause any _problem_ for it to be here its just, pointless for it to be here